app/Controllers/tagController.php

```diff
@@ -90,7 +90,7 @@ public function addAction() {
 
 		$name = Minz_Request::param('name');
 		$tagDAO = FreshRSS_Factory::createTagDao();
-		if (null === $tagDAO->searchByName($name)) {
+		if (null === $tagDAO->searchByName($name) && strlen($name) > 0) {
```

Comment:
If probably makes sense to test the variable before using it rather than after
```suggestion
		if (strlen($name) > 0 && null === $tagDAO->searchByName($name)) {
```